config-linux.md

```diff
@@ -189,6 +189,98 @@ In addition to any devices configured with this setting, the runtime MUST also s
 * [`/dev/ptmx`][pts.4].
   A [bind-mount or symlink of the container's `/dev/pts/ptmx`][devpts].
 
+## <a name="configLinuxNetworkDevices" />Network Devices
+
+Linux network devices are entities that send and receive data packets.
+They are not represented as files in the /dev directory, unlike block devices, network devices are represented with the [`net_device`][net_device] data structure in the Linux kernel.
+Network devices have their own network namespace and a set of operations distinct from regular file operations. Examples of network devices include Ethernet cards, loopback devices, and virtual devices like bridges, VLANs, and MACVLANs.
```

Comment:
> Network devices have their own network namespace

Well, they don't (but I'm not sure how to phrase it correctly -- something like "Any network device belongs to a [particular] network namespace [at any time]" -- with the parts in square brackets being highly optional)

Reply:
how about `belongs to exactly one network namespace at any given time`